### DISCUSSION TITLE
Stuck source reconciliation

### DISCUSSION THREAD

**Discussion by dimitrigeo** [fluxcd/flux2#2845] [General]
Hi!
I wondering if i can get some pointers, I am very new to fluxcd and am working on a issue with one namespace and kustomiztion that just will not update the revision, while all the other revisions on the same cluster are updating fine. the namespace handles jobs and cronjobs.

I have tried to manually reconcile the source but I just get this:
```

`╰─ flux reconcile source git cypress                                                                                                                                                               ─╯
✗ gitrepositories.source.toolkit.fluxcd.io "cypress" not found`
```

```
╰─ flux version                                                                                                                                                                                    ─╯
flux: v0.31.1
helm-controller: v0.22.1
kustomize-controller: v0.26.1
notification-controller: v0.24.0
source-controller: v0.25.5

╰─ flux check                                                                                                                                                                                      ─╯
► checking prerequisites
✔ Kubernetes 1.22.6 >=1.20.6-0
► checking controllers
✔ helm-controller: deployment ready
► ghcr.io/fluxcd/helm-controller:v0.22.1
✔ kustomize-controller: deployment ready
► ghcr.io/fluxcd/kustomize-controller:v0.26.1
✔ notification-controller: deployment ready
► ghcr.io/fluxcd/notification-controller:v0.24.0
✔ source-controller: deployment ready
► ghcr.io/fluxcd/source-controller:v0.25.5
✔ all checks passed
```

<img width="311" alt="Screenshot 2022-06-15 at 15 46 21" src="https://user-images.githubusercontent.com/11459620/173843032-874ae12a-f467-4f4a-83ef-ef42a8ca6c66.png">

**Comment by kingdonb** [comment]
There is not much information to go on here, without the manifests that you used it will be hard to narrow down what has gone wrong.

If you run `kubectl describe gitrepo <theRepo>` you should get more information about what is wrong with the resource if it has failed.

Based on the error you posted, it looks as if you are maybe not looking in the right namespace. Try adding `-n namespace` to access the resource in the namespace where it is located.
